Makefile

```diff
@@ -33,9 +33,19 @@ endif
 ifndef BUILD_TIME
 	$(eval BUILD_TIME=$(shell date +%Y-%m-%d_%H:%M:%S_%Z))
 endif
-	sudo docker build -t $(BINARY_NAME_AKO):latest --label "BUILD_TAG=$(BUILD_TAG)" --label "BUILD_TIME=$(BUILD_TIME)" -f Dockerfile.ako .
 
+ifdef GOLANG_SRC_REPO
+	$(eval BUILD_ARG_GOLANG=--build-arg golang_src_repo=$(GOLANG_SRC_REPO))
+else
+	$(eval BUILD_ARG_GOLANG="")
```

Comment:
This line needs to be `$(eval BUILD_ARG_GOLANG=)` so as to support the workflow where GOLANG_SRC_REPO is not exported as an environment variable

Reply:
Okay I followed your patch file. Done now.

Reply:
12:23:24  ---> Running in 76435a1e5de0
12:23:24 Removing intermediate container 76435a1e5de0
12:23:24  ---> bd00a098c783
12:23:24 Step 9/15 : FROM ${photon_src_repo}
12:23:25 base name (${photon_src_repo}) should not be blank
12:23:25 Makefile:33: recipe for target 'docker' failed
12:23:25 make: *** [docker] Error 1
12:23:25 Build step 'Execute shell' marked build as failure
12:23:25 Started calculate disk usage of build
12:23:25 Finished Calculation of disk usage of build in 0 seconds 

Looks it does not work.

Reply:
Missed your comment below.